apps/pie-docs/src/patterns/overlay-patterns/guidance/guidance.md

```diff
@@ -0,0 +1,386 @@
+---
+eleventyNavigation:
+    key: Guidance
+    parent: 'Overlay Patterns'
+    order: 2
+eleventyComputed:
+    zIndexes: "{% include './z-indexes.json' %}"
+shouldShowContents: true
+---
+
+## Decision tree
+
+You can follow this decision tree to choose the most appropriate UI component based on the specific context, importance, and user interaction requirements of your product interface.
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/overlay-patterns/decision-tree.svg",
+  mobileSrc:"../../../../assets/img/patterns/overlay-patterns/decision-tree-narrow.svg",
+  alt: "A decision tree to choose the most appropriate UI component based on the specific context.",
+  width: "1440px"
+} %}
+
+---
+
+## Full width notification
+
+A full width notification is a visual element that appears at the top of the screen. It's commonly used to display important messages, announcements or alerts.
+
+{% notification {
+  type: "information",
+  iconName: "info-circle",
+  message: "Check out our [full documentation](https://pie.design/components/notification/) for this component."
+} %}
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/overlay-patterns/banner.svg",
+  mobileSrc:"../../../../assets/img/patterns/overlay-patterns/banner-narrow.svg",
+  alt: "The error variant of the full width notification component that contains heading, one line of text and a close icon to dismiss.",
+  width: "872px"
+} %}
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+            "Use them at the top of the screen, only under the header.",
+            "Use them for succinct messaging.",
+            "They are triggered by the system."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Don’t use them to display long form content.",
+            "Don’t use them for non-essential information."
+        ]
+    }
+} %}
```

Comment:
Does this need to be aligned with the do and dont's of the [notification component page](https://pr1843-docs.pie.design/components/notification/#dos-and-donts)?

Reply:
Same question for all the other components on this page

Reply:
These descriptions do not have to be aligned with the `do` and `dont's `on the components home page, but they do not contradict them.